content/authors/sarah-gibson/_index.md

```diff
@@ -0,0 +1,61 @@
+---
+# Display name
+title: "Sarah Gibson"
+
+# Username (this should match the folder name and the name on publications)
+authors:
+- "sarah-gibson"
+
+# Is this the primary user of the site?
+superuser: false
+
+# Role/position (e.g., Professor of Artificial Intelligence)
+role: Open Source Infrastructure Engineer
+
+# Organizations/Affiliations
+organizations:
+- name: 2i2c
+  url: "https://2i2c.org"
+- name: Project Jupyter
+  url: "https://jupyter.org"
+- name: The Turing Way
+  url: "https://the-turing-way.netlify.app/"
+
+# Short bio (displayed in user profile at end of posts)
+bio:
+
+# List each interest with a dash
+# interests:
+# -
+
+# Social/Academic Networking
+# For available icons, see: https://sourcethemes.com/academic/docs/page-builder/#icons
+#   For an email link, use "fas" icon pack, "envelope" icon, and a link in the
+#   form "mailto:your-email@example.com" or "#contact" for contact widget.
+social:
+- icon: envelope
+  icon_pack: fas
+  link: 'mailto:sgibson@2i2c.org'
+- icon: twitter
+  icon_pack: fab
+  link: https://twitter.com/drsarahlgibson
+- icon: github
+  icon_pack: fab
+  link: https://github.com/sgibson91
+
+# Enter email to display Gravatar (if Gravatar enabled in Config)
+# email: ""
+
+# Organizational groups that you belong to (for People widget)
+#   Set this to `[]` or comment out if you are not using People widget.
+user_groups:
+- Open Engineering Team
+---
+
+Sarah Gibson is an Open Source Infrastructure Engineer at 2i2c, an open source contributor and advocate.
+She holds more than 2 years of experience as a Research Engineer at a national institute for data science and artificial intelligence, as well as holding a core contributor role in the open source projects [Binder](https://jupyter.org/binder), [JupyterHub](https://jupyter.org/hub), and [_The Turing Way_](https://the-turing-way.netlify.app/).
+Sarah is passionate about working with domain experts to leverage cloud computing in order to accelerate cutting-edge, data-intensive research and disseminating the results in an open, reproducible and reusable manner.
+
+Sarah holds a Fellowship with the [Software Sustainability Institute](https://software.ac.uk) and advocates for best software practices in research.
+She is a member of the [mybinder.org operating team](https://jupyterhub-team-compass.readthedocs.io/en/latest/team.html#jupyterhub-team) and maintains infrastructure supporting over 150k launches of reproducible computational environments per week.
+She has also mentored projects through 2 cohorts of the [Open Life Science](https://openlifesci.org) programme, imparting lived experience of her skills participating and leading in open science projects.
```

Comment:
```suggestion
She has also mentored projects through two cohorts of the [Open Life Science](https://openlifesci.org) programme, imparting lived experience of her skills participating and leading in open science projects.
```

Reply:
I suggested an edit to follow the standard @consideRatio suggests, but I don't feel strongly about it, I think we could merge either way. @sgibson91 do you have a preference?

Reply:
Warning - there is another sentence with a `2` =)

Sorry for nitpicking!

Reply:
At least I followed the consistent part of the rule! 😆 I have no preference either way, feel free to use letters if that's the standard we'd like to have on the site 🙂